demo/fiber/iotdb-sync-stress-demo/pom.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.example</groupId>
+    <artifactId>iotdb-sync-stress-demo</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+    <properties>
+        <maven.compiler.source>8</maven.compiler.source>
+        <maven.compiler.target>8</maven.compiler.target>
+    </properties>
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.iotdb</groupId>
+            <artifactId>iotdb-session</artifactId>
+            <version>0.12.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.iotdb</groupId>
+            <artifactId>iotdb-jdbc</artifactId>
+            <version>0.12.0</version>
+        </dependency>
+    </dependencies>
```

Comment:
there is no jar-with-dependencies configure in pom,  target/iotdb-sync-stress-demo-1.0-SNAPSHOT-jar-with-dependencies.jar will not be generated.

Reply:
@huishi-hs Thanks for you review!

I have updated the new pom file and will generate iotdb-sync-stress-demo-1.0-SNAPSHOT-jar-with-dependencies.jar.